packages/button/src/component.tsx

```diff
@@ -15,73 +15,63 @@ import type { ButtonProps } from './props.js';
 const buttonVariants = ['primary', 'secondary', 'negative', 'utility', 'pill', 'link'] as const;
 
 export const Button = forwardRef<HTMLButtonElement | AnchorHTMLAttributes<HTMLAnchorElement>, ButtonProps>((props, ref) => {
-  const { primary, secondary, negative, utility, quiet, small, link, pill, loading, fullWidth, ...rest } = props;
+  const { primary, secondary, negative, utility, quiet, small, link, href, pill, loading, disabled, fullWidth, ...rest } = props;
```

Comment:
Not sure which change caused this but now you can tab to a disabled button, which was not possible before and I think should not be possible.
<img width="364" alt="Screenshot of an outlined disabled button" src="https://github.com/user-attachments/assets/2bfbd445-93ee-4801-941a-b3103f1a1a9b">

Reply:
Good catch! I'll look into it! :)

Reply:
Updated in this commit: [78a2d55](https://github.com/warp-ds/react/pull/274/commits/78a2d550d475e34a957476b450aa330242483305)